### PR TITLE
Updated floweditor 

### LINKED ISSUE #1696: Unable to click [Ok] on wait for response modal
**Describe the bug**
While creating a new `wait for response` node, our users were unable to click on the Ok button. the pattern was kind of as shown in the image below

choosing has any words (leave empty), enter category, enter result name and click ok.

Screenshot 2021-10-08 at 12 24 21 PM

### PR DESCRIPTION
**Pull request by mdshamoon**
## Summary

- Removed name field from update contact node
- Updated floweditor to v1.15 that fixes #1696

### CODE REVIEW

File: src/config/absinthe.ts [hunk 1]
```diff
@@ -49,8 +49,11 @@ socketConnection.onError(async (error: any) => {
   if (connectionFailureCounter >= CONNECTION_RECONNECT_ATTEMPTS) {
     socketConnection.disconnect();
 
-    // let's trigger the reconnect function after 5sec
-    setTimeout(() => resetWSConnection(socketConnection), 5000);
+    // lets not indefinitely try to reconnect
+    if (connectionFailureCounter < WEBSOCKET_RECONNECTION_ATTEMPTS) {
```
Comment (kurund) [inline src/config/absinthe.ts:53]: @mdshamoon 

We only attempt to connect  `CONNECTION_RECONNECT_ATTEMPTS` times, ie. 5 times. So not sure why we need this.

If this is not linked to floweditor changes maybe move these changes to separate PR.

Comment (mdshamoon) [reply src/config/absinthe.ts]: The condition we put for the CONNECTION_RECONNECT_ATTEMPTS does not stop reconnecting the WebSocket connection, it goes on indefinitely. 

I will create a separate issue and PR for this since these changes are not related to flow editor 
I added in this PR because we were getting logged out in the preview due to this. We can discuss this further in the call.

**Review by kurund — APPROVED** [review]
looks good.
